### PR TITLE
Feat: slideout on form page

### PR DESCRIPTION
**Pull request by AndrewBB-BBD**
Please test the slide out using different window sizes! :)

Click "What is required of me?" to open it. 

I also added a bit of modal logic. If you click "Contact me" then the modal will appear. You'll need to refresh the page to get rid of it.

### CODE REVIEW

File: Client/adoptForm/adoptForm.html [hunk 1]
```diff
@@ -49,9 +51,54 @@
         ></textarea
         ><br /><br />
         <button class="backButton" onclick="history.back()">Back</button>
-        <button class="submitButton">Contact me</button>
+        <button class="submitButton" id="contactMeButton">Contact me</button>
```
Comment (MaddieLymei) [inline Client/adoptForm/adoptForm.html]: Maybe rename to "Apply" as the name. 

Also, "Go Home" after submission needs an action to show it is being hovered over or clicked.

File: Client/adoptForm/adoptForm.js [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+const successModal = document.getElementById("successModal");
+const failureModal = document.getElementById("failureModal");
+const contactMeButton = document.getElementById("contactMeButton");
 
-function validateForm() {
-    
-}
\ No newline at end of file
+const telPattern = (pattern = "[0-9]{3}-[0-9]{2}-[0-9]{3}");
+function validateForm() {}
```
Comment (MaddieLymei) [inline Client/adoptForm/adoptForm.js]: More form validation required.

File: Client/adoptForm/adoptForm.css [hunk 3]
```diff
@@ -133,7 +134,80 @@
   margin-bottom: 5%;
 }
 
-@media all and (max-width: 750px) {
+/* Slideout CSS */
+.slideout {
```
Comment (MaddieLymei) [inline Client/adoptForm/adoptForm.css]: Possible to have contact details of shelter on the LHS with the dogs name? Something to show what they are applying for?

File: Client/adoptForm/adoptForm.css [hunk 4]
```diff
@@ -98,6 +98,7 @@
   overflow: auto;
```
Comment (MaddieLymei) [inline Client/adoptForm/adoptForm.css]: css needs to be in the public folder/css/

File: Client/adoptForm/adoptForm.html [hunk 5]
```diff
@@ -17,7 +17,9 @@
     <main class="mainBody">
       <form action="" class="adoptionForm">
         <header>Take the first step to take your fluffy friend home!</header>
-        <button class="sliderLink">What is required of me?</button><br /><br />
+        <button class="sliderLink" id="openSlideoutButton">
```
Comment (MaddieLymei) [inline Client/adoptForm/adoptForm.html]: changes need to be put in views/adoptForm.html, watch out for my changes and work with them :-)

File: Client/adoptForm/adoptForm.js [hunk 6]
```diff
@@ -1,5 +1,41 @@
-const telPattern = pattern="[0-9]{3}-[0-9]{2}-[0-9]{3}";
```
Comment (MaddieLymei) [inline Client/adoptForm/adoptForm.js:1]: public/js/adoptForm.js needs these changes with my changes I have inserted please

**Review by nikitaabrey — APPROVED** [review]
I really like the slide a lot it works on all the screen sizes for me and works smoothly. Thank you so much for all the effort on it @AndrewBB-BBD!! 

Could the modal maybe in the center of the screen?

**Review by Natasha-Khumalo-BBD — APPROVED** [review]

**Review by MaddieLymei — CHANGES_REQUESTED** [review]

**Review by MaddieLymei — APPROVED** [review]

**Pull request merged**